tests/raw_to_standard/navjoy_568_test.py

```diff
@@ -0,0 +1,609 @@
+                                -103.17889641706886,
+                                40.61979008921054
+                            ]
+                        ]
+                    ],
+                },
+            ],
+            "reductionJustification": "Crews roadside.",
+
+            "streetNameFrom2": "US-34",
+            "directionOfTraffic2": " East/West ",
+            "requestedTemporarySpeed2": "45",
+            "workStartDate2": "2021-08-09T13:00:00.000Z",
+            "workEndDate2": "2021-09-24T23:00:00.000Z",
+            'currentPostedSpeed2': None,
+            'mileMarkerEnd2': None,
+            'mileMarkerStart2': None,
+        }
+    }
+
+    actual = navjoy_568.expand_speed_zone(event)
+
+    assert expected_results.test_expand_speed_zone_2_expected == actual
+
+
+# I hate how long this test is, but this is what is has to be to test all 4 at the same time
+def test_expand_speed_zone_2_3_4():
+    event = {
+        "sys_gUid": "Form568-cb0fdaf0-c27a-4bef-aabd-442615dfb2d6",
+        "data": {
+            "srzmap": [
+            ],
+            "reductionJustification": "Crews roadside.",
+            "streetNameFrom": "US-34",
+            "directionOfTraffic": " East/West ",
+            "requestedTemporarySpeed": "1",
+            "workStartDate": "2021-08-09T13:00:00.000Z",
+            "workEndDate": "2021-09-24T23:00:00.000Z",
+            'currentPostedSpeed': None,
+            'mileMarkerEnd': None,
+            'mileMarkerStart': None,
+
+            "streetNameFrom2": "US-34",
+            "directionOfTraffic2": " East/West ",
+            "requestedTemporarySpeed2": "2",
+            "workStartDate2": "2021-08-09T13:00:00.000Z",
+            "workEndDate2": "2021-09-24T23:00:00.000Z",
+            'currentPostedSpeed2': None,
+            'mileMarkerEnd2': None,
+            'mileMarkerStart2': None,
+
+            "streetNameFrom3": "US-34",
+            "directionOfTraffic3": " East/West ",
+            "requestedTemporarySpeed3": "3",
+            "workStartDate3": "2021-08-09T13:00:00.000Z",
+            "workEndDate3": "2021-09-24T23:00:00.000Z",
+            'currentPostedSpeed3': None,
+            'mileMarkerEnd3': None,
+            'mileMarkerStart3': None,
+
+            "streetNameFrom4": "US-34",
+            "directionOfTraffic4": " East/West ",
+            "requestedTemporarySpeed4": "4",
+            "workStartDate4": "2021-08-09T13:00:00.000Z",
+            "workEndDate4": "2021-09-24T23:00:00.000Z",
+            'currentPostedSpeed4': None,
+            'mileMarkerEnd4': None,
+            'mileMarkerStart4': None,
+        }
+    }
+
+    actual = navjoy_568.expand_speed_zone(event)
+
+    assert expected_results.test_expand_speed_zone_2_3_4_expected == actual
+
+
+@patch('uuid.uuid4')
+def test_generate_standard_messages_from_string(mockuuid):
+    uuid.uuid4 = Mock()
+    uuid.uuid4.side_effect = ['we234de', '23wsg54h', '7fa1dfas', '23h327j']
+    xml_string = """[
```

Comment:
I'm not sure at what size of example data we want to load from a separate file, but this one would be a good candidate.